### PR TITLE
Improve flash-attn mask calculation for auto llama

### PR DESCRIPTION
**Pull request by From00**
### PR types
Performance optimization

### PR changes
Others

### Description

消除静半组网中，在开fused_flash_attention情况下对attention_mask的冗余计算。

### CODE REVIEW

**Review by haohongxiang — APPROVED** [review]
LGTM

**Review by zhaoyinglia — APPROVED** [review]
LGTM
